REFERENCE.md

```diff
@@ -187,6 +187,26 @@ wireguard::interface { 'wg0':
 }
 ```
 
+##### Peer with one node, setup dualstack firewall rules and RoutingPolicyRule
```

Comment:
That should go into https://github.com/voxpupuli/puppet-wireguard/blob/master/manifests/interface.pp see, all the existing `@example` entries there, the generator will put it into the REFERENCE.md then